masterlist.yaml

```
@@ -826,17 +826,24 @@ plugins:
       - crc: 0xB362BA90
         util: 'FO3Edit v4.0.3h (Hotfix 1)'
 
-  - name: 'CALIBR.esm'
+
+  - name: '(CALIBR(xMerchant|)\.esp)'
```

Removed the extra newline.
You don't need to wrap the filename generally, I can't recall it being used.
`?` can be placed after a capture group, to find if something occurs once or not at all.
`[]` can be used to capture one of any characters listed.
```suggestion
  - name: 'CALIBR(xMerchant)?\.es[mp]'
```